tests/lib.rs

```diff
@@ -21,15 +22,15 @@ fn test_gzip_and_zlib_encoded_and_raw_are_the_same() {
     assert_eq!(z, c);
     assert_eq!(z, zstd);
 
-    if let LayerData::Finite(tiles) = &c.layers[0].tiles {
+    if let LayerData::Finite(tiles) = &c.layers[0].layer_type.as_tile_layer().unwrap().tiles {
```

Comment:
Hmm, what if we would put `as_tile_layer` on the `Layer`, so that we could get `&c.layers[0].as_tile_layer()`?

Reply:
Oops, saw this too late

Reply:
Imo not a good decision because `as_tile_layer` only borrows `LayerType`, not the entire `Layer`

Reply:
Ah, this type of consideration is still new to me. :-)